zebrad/tests/acceptance.rs

```diff
@@ -3603,6 +3607,162 @@ async fn nu6_funding_streams_and_coinbase_balance() -> Result<()> {
     Ok(())
 }
 
+/// Test successful block template submission as a block proposal.
+///
+/// This test can be run locally with:
+/// `cargo test --package zebrad --test acceptance --features getblocktemplate-rpcs -- nu7_nsm_transactions --exact --show-output`
+#[tokio::test(flavor = "multi_thread")]
+#[cfg(all(feature = "getblocktemplate-rpcs", feature = "tx_v6"))]
+async fn nu7_nsm_transactions() -> Result<()> {
+    use zebra_chain::{
+        chain_sync_status::MockSyncStatus,
+        parameters::testnet::{self, ConfiguredActivationHeights, ConfiguredFundingStreams},
+        serialization::ZcashSerialize,
+        work::difficulty::U256,
+    };
+    use zebra_network::address_book_peers::MockAddressBookPeers;
+    use zebra_node_services::mempool;
+    use zebra_rpc::client::HexData;
+    use zebra_test::mock_service::MockService;
+    let _init_guard = zebra_test::init();
+
+    tracing::info!("running nu7_nsm_transactions test");
+
+    let base_network_params = testnet::Parameters::build()
+        // Regtest genesis hash
+        .with_genesis_hash("029f11d80ef9765602235e1bc9727e3eb6ba20839319f761fee920d63401e327")
+        .with_target_difficulty_limit(U256::from_big_endian(&[0x0f; 32]))
+        .with_disable_pow(true)
+        .with_slow_start_interval(Height::MIN)
+        .with_activation_heights(ConfiguredActivationHeights {
```

Comment:
The `ParametersBuilder` returned by `testnet::Parameters::build()` initially has all of the default Testnet values, but the deferred pool balance may be less on a configured Testnet at NU6.1 activation than it would be on the default Testnet. For this Testnet, the deferred pool balance would be zero when NU6.1 activates, so the lockbox disbursements must be zero as well.

Using Regtest instead could work too.


```suggestion
        .with_lockbox_disbursements(vec![])
        .with_activation_heights(ConfiguredActivationHeights {
```